src/App/Fossa/ArchiveUploader.hs

```diff
@@ -91,7 +91,8 @@ archiveUploadSourceUnit baseDir apiOpts vendoredDeps = do
 
   -- archiveBuildUpload takes archives without Organization information. This orgID is appended when creating the build on the backend.
   -- We don't care about the response here because if the build has already been queued, we get a 401 response.
-  _ <- Fossa.archiveBuildUpload apiOpts (ArchiveComponents $ NonEmpty.toList archives)
+  res <- traverse (Fossa.archiveBuildUpload apiOpts) (NonEmpty.toList archives)
+  logDebug $ pretty $ show res
```

Comment:
[Question] This shows `X-Amz-Credential=...`, but I'm presuming this is a pre-signed URL for put object with a specific scope on such object so this is fine?

Reply:
Yea this will be the information for the pre-signed URL, which is helpful when figuring out exactly where the CLI tried to upload the archive. I think exposing this only for debug logs is acceptable unless you have a further objection.

Reply:
No objections - I wanted to make sure this was not a security gotcha!